### ISSUE TITLE
RangeError ?

### ISSUE THREAD

**Issue by forumapp28** [vram4/Ogar#125]
http://prntscr.com/7mnw26

See pictures.

**Comment by vram4** [comment]
Its fine, anyone that causes a range error gets kicked immediately.

**Comment by forumapp28** [comment]
I do not understand ?

**Comment by forumapp28** [comment]
Is there no solution ?

**Comment by themuddfamily2** [comment]
Its not a bug, the actual error appearing means that it has stopped a crash.

**Comment by Jackster35** [comment]
Yes, the code has caught the crash and prevented it from killing your server completely. It's something out of vram4's hands I believe so this is the best solution. (Plus, it doesn't change anything)
